docs/guide/src/node/pd/chain-upgrade.md

```diff
@@ -64,10 +63,8 @@ and provide that value in the documentation. Or should we just pick a genesis st
 -->
 5. Apply the migration: `pd migrate --genesis-start "{{ #include ../../upgrade_genesis_time_70_71.md }}" --target-directory ~/.penumbra/testnet_data/node0/pd-exported-state/ --migrate-archive ~/.penumbra/testnet_data/node0/pd-migrated-state-{{ #include ../../penumbra_version.md }}.tar.gz`.
    You must use that precise genesis time, otherwise your node will not be able to reach consensus with the rest of the network.
-6. Move the migrated state into place: `mkdir ~/.penumbra/testnet_data/node0/pd && mv ~/.penumbra/testnet_data/node0/pd-exported-state/rocksdb ~/.penumbra/testnet_data/node0/pd/`
-7. Move the upgrade cometbft state into place: `cp ~/.penumbra/testnet_data/node0/pd-exported-state/genesis.json ~/.penumbra/testnet_data/node0/cometbft/config/genesis.json
-   && cp ~/.penumbra/testnet_data/node0/pd-exported-state/priv_validator_state.json ~/.penumbra/testnet_data/node0/cometbft/data/priv_validator_state.json`
-8. Then we clean up the old CometBFT state: `find ~/.penumbra/testnet_data/node0/cometbft/data/ -mindepth 1 -maxdepth 1 -type d -exec rm -r {} +`
+6. Move the migrated state into place: `rm -r ~/.penumbra/testnet_data/node0/pd/rocksdb && mv ~/.penumbra/testnet_data/node0/pd-exported-state/rocksdb ~/.penumbra/testnet_data/node0/pd/`
+7. Move the new cometbft genesis into place: `cp ~/.penumbra/testnet_data/node0/pd-exported-state/genesis.json ~/.penumbra/testnet_data/node0/cometbft/config/genesis.json`
```

Comment:
You must reset the comet WAL, blockstore, and evidence DB

Reply:
Comet won't start if you don't do the WAL/blockstore, and it will corrupt your state if you don't do the evidence db if i recall correctly

Reply:
Thanks for explaining. Substantively, that means we should recommend either:

1. `find ~/.penumbra/testnet_data/node0/cometbft/data/ -mindepth 1 -maxdepth 1 -type d -exec rm -r {} +`, as before; or
2. `cometbft --home ~/.penumbra/testnet_data/node0/cometbft/ reset-state`

AIUI, the two are functionally equivalent. If so, I'd prefer to recommend the latter.

Reply:
I think `reset-state` is what we want, I can test it later today

Reply:
We should check that it doesn't delete the indexing KV, will check

Reply:
See testing report below, @erwarnor. I believe the revised docs are what we need, but would certainly appreciate a +1 from you!